docs/how-to-contribute/dev-onboarding.md

```diff
@@ -0,0 +1,126 @@
+---
+sidebar_position: 4
+title: "Dev Onboarding"
+sidebar_label: "Dev Onboarding"
+---
+
+## General Checklist
+:::info 
+Please make sure to go through the [General Onboarding](./general-onboarding.md) first.
+:::
+
+## New Devs
+In case you aren't familiar with working on collaborative projects on GitHub, you could watch this video: [Git & GitHub: GitHub Workflow for Open Source](https://www.youtube.com/watch?v=4VY0kHqIqyU), and read this article:[How to Contribute to an Open Source Project](https://css-tricks.com/how-to-contribute-to-an-open-source-project/).
+
+We also recommend watching this one: [Git for Professionals Tutorial - Tools & Concepts for Mastering Version Control with Git
+](https://www.youtube.com/watch?v=Uszj_k0DGsg) to understand how fork, branch and pull requests work.
+
+Please make sure you understand the following:
+- [ ] GitHub: be able to fork a repo and submit a pull-request
+- [ ] GitHub: be able to rebase your branch with remote
+
+Now you have learned the basics, let's get our hands dirty! 
+
+## Challenge 1: Git and GitHub Best Practices
+
+Suitable for: all devs
+
+Estimated time: 20 mins
+
+Following the instructions to create your first pull request in our documentation repo.
+
+
+1. Fork and clone the [Openbeta documentation repo](https://github.com/OpenBeta/docs.openbeta.io/tree/develop)
+
+    `git clone url-of-your-forked-repo`
+
+2. Run `yarn install` to install all the dependencies. Then run `yarn start` to run the project locally. 
+
+3. Create a new branch in Git.
+
+    `git checkout -b add-contributor-yun`
+
+Replace *yun* with your name.
+
+
+4. Update your fork with the latest Openbeta changes. 
+- Add Openbeta as an upstream remote. 
+    
+    `git remote add openbeta https://github.com/OpenBeta/docs.openbeta.io.git`
+- Fetch the latest changes from Openbeta
+    
+    `git fetch openbeta develop`
+- In the branch are working on, update the changes.
+    
+    `git merge openbeta/develop`
```

Comment:
`git rebase openbeta/develop` is more consistent with the tip below and yields a cleaner history.  Can be more confusing though...

Reply:
yeah I was debating on which one to use here...